L-Italia-tra-le-due-guerre-il-fascismo.md

```diff
@@ -163,23 +162,23 @@ Uno dei primi provvedimenti in ambito economico fu la cosiddetta *`battaglia del
 - La battaglia del grano ebbe successo a scapito dell'allevamento e delle colture specializzate rivolte all'esportazione
-- L'Italia era ancora un Paese povero di materie prime e questo portò ad un ulteriore indebolimento del sistema produttivo
+- L'Italia era ancora un Paese povero di materie prime e questo portò ad un ulteriore indebolimento del sistema produttivo, sopratutto sul campo industriale
 
 Nel `1927` venne introdotta la `Carta del lavoro` che stabiliva l'obbligo per tutti i settori di organizzarsi in corporazioni composte da lavoratori e padroni e soggette ad un apposito ministero. Questa ideologia fu chiamata `corporativismo`, che inoltre prevedeva l'impossibilità di azioni sindacali da parte di socialisti, comunisti e cattolici. Questo però non funzionò mai e portò vantaggi solamente agli imprenditori che potevano tenere basso il costo del lavoro e influenzare le decisioni politiche.
 
 Per fronteggiare gli effetti della [`crisi economica del 1929`][crisi-1929] lo Stato intervenne sempre più massicciamente durante gli Anni Trenta, creando l'`Istituto Mobiliare Italiano` (IMI) per sostenere le industrie in difficoltà e l'`Istituto per la Ricostruzione Industriale` (IRI) che divenne azionista di maggioranza di alcune banche e prese il controllo di alcune grandi aziende italiane.\
 Queste misure di risanamento dovevano essere provvisorie in attesa di privatizzare nuovamente le imprese, ma in realtà divenne un ente permanente dal 1937. Iniziò allora la moltiplicazione degli enti pubblici tuttora esistenti caratterizzati da una gestione pesantemente burocratica con scarsa efficienza, ma molto rilevante per quanto riguarda la forza economica e politica.
 
-Il regime fascista era stato caratterizzato sin dagli inizi da una forte componente nazionalista, presentandosi Mussolini come il condottiero (*duce*) che avrebbe risollevato l'Italia dalle penalizzazioni subite dopo la fine della prima guerra mondiale.\
-Ciò nonostante i proclami nazionalisti rimasero sempre molto vaghi, preferendo Mussolini la tradizionale amicizia con Francia e Inghilterra. Ma questo cambiò nel `1934` con la decisione di conquistare l'`Etiopia`, volendo così espandere le colonie italiane e offrire una meta per l'emigrazione italiana per quindi risollevare la situazione economica.
+Il regime fascista era stato caratterizzato sin dagli inizi da una forte componente nazionalista, presentando Mussolini come il condottiero (*duce*) che avrebbe risollevato l'Italia dalle penalizzazioni subite dopo la fine della prima guerra mondiale.\
+Ciò nonostante i proclami nazionalisti rimasero sempre molto vaghi, preferendo Mussolini alla tradizionale amicizia con Francia e Inghilterra. Ma questo cambiò nel `1934` con la decisione di conquistare l'`Etiopia`, volendo così espandere le colonie italiane e offrire una meta per l'emigrazione italiana per quindi risollevare la situazione economica.
```

Comment:
Sicuro che serva questa modifica?
In quanto Mussolini preferiva la tradizionale amicizia ai proclami nazionalisti, non il contrario.

Volendo si potrebbe modificare, per rendere più chiaro magari, in "... in quanto Mussolini preferiva la tradizionale ...".

Reply:
Eh, fai conto che lo avevo modificato cosi perché pensavo fosse un typo (._.). Si,  ti direi che l'ultima proposta che hai fatto può andare